billing/test/lib/egress-traffic.js

```diff
@@ -101,9 +121,36 @@ export const test = {
     } finally {
       if (stripeCustomerId) {
         // 5. Delete the test customer from stripe
-        const deletedCustomer = await ctx.stripe.customers.del(stripeCustomerId);
+        const deletedCustomer = await ctx.stripe.customers.del(stripeCustomerId)
         assert.ok(deletedCustomer.deleted, 'Error deleting customer from stripe')
       }
     }
   }
+}
+
+/**
+ * Retry a function with exponential backoff
+ * 
+ * @param {Function} fn - The function to retry
+ * @param {number} maxRetries - Maximum number of retries
+ * @param {number} initialDelay - Initial delay in milliseconds
+ * @returns {Promise<any>} - The result of the function if successful
+ */
+async function retryWithExponentialBackoff(fn, maxRetries, initialDelay) {
```

Comment:
we use the `p-retry` library for this in a lot of places, would that work here?

Reply:
Good call. That works. I just updated the code with the `p-retry` lib. Thanks for flagging.